src/Stomp/Client.php

```diff
@@ -154,6 +160,28 @@ public function setVhostname($host = null)
         $this->host = $host;
     }
 
+  /**
+   * Set the desired heartbeat for the connection.
+   *
+   * A heartbeat is a specific message that will be send / received when no other data is send / received
+   * within an interval - to indicate that the connection is still stable. If client and server agree on a beat and
+   * the interval passes without any data activity / beats the connection will be considered as broken and closed.
+   *
+   * If you define a heartbeat, you must assure that your application will send data within the interval.
```

Comment:
`jein` - without a heartbeat declaration you can send data whenever you want, you could even add some minutes of `sleep` between the communication.
With heartbeats enabled you must do `something` within the interval. 
The emitter is not automatically attached to the connection. Maybe this description is not yet good enough yet. But it's not that easy, hm...